pom.xml

```diff
@@ -1460,6 +1461,7 @@
         <groupId>com.google.guava</groupId>
         <artifactId>guava-testlib</artifactId>
         <version>${guava.version}</version>
+        <type>test-jar</type>
         <scope>test</scope>
```

Comment:
Why are we setting the `<type>` here (it doesn't seem to be consumed in any pom.xml)?

Also, we are duplicating the definition of guava-testlib around line 1400 in this pom.xml so we should be able to remove this entire definition.

Reply:
You are right, I thought the warning was related to the type and not the duplicated dependency. Removing it should make it.